### PR TITLE
feat(sign-up): siret create a "corporation not created yet" path

### PR DESCRIPTION
**Pull request by dectotam**
ref: MANAGER-12321

| Question         | Answer
| ---------------- | ---
| Branch?          | master
| Bug fix?         | no
| New feature?     | no
| Breaking change? | no
| Tickets          | Feat MANAGER-12321
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] ~~Breaking change is mentioned in relevant commits~~

## Description

Add possibility to say that corporation has not been created yet, so user can create account without providing a siret

### CODE REVIEW

**Review by qpavy — DISMISSED** [review]

**Review by darsene — DISMISSED** [review]

File: packages/manager/modules/sign-up/src/form/activity/activity.html [hunk 1]
```diff
@@ -137,7 +137,38 @@
...
+            data-on-change="$ctrl.resetCorporationData()"
+            data-label="{{:: 'sign_up_activity_corporation_not_created_title' | translate }}"
+            data-values="[false]"
+        >
+            <oui-select-picker-description>
+                <span
+                    data-translate="sign_up_activity_corporation_not_created_description"
+                ></span>
+            </oui-select-picker-description>
+        </oui-select-picker>
```
Comment (ghyenne) [inline packages/manager/modules/sign-up/src/form/activity/activity.html:168]:
```suggestion
<div class="mb-4">
        <oui-select-picker
            data-ng-repeat="value in [true, false] track by $index"
            data-model="$ctrl.corporationIsCreated"
            data-required
            data-label="{{:: `sign_up_activity_corporation_created_${value}_title` | translate }}"
            data-values="[value]"
        >
            <oui-select-picker-description>
                <span
                    data-translate="{{ 'sign_up_activity_corporation_created_'+ value +'_title'}}"
                ></span>
            </oui-select-picker-description>
        </oui-select-picker>
</div>
```

File: packages/manager/modules/sign-up/src/form/activity/translations/Messages_fr_FR.json [hunk 2]
```diff
@@ -9,6 +9,10 @@
   "sign_up_activity_field_vat_personalcorporation_more": "Numéro d'identification d'entreprise ({{ vatLabel }})",
   "sign_up_activity_field_organisation_association": "Nom de l'association",
   "sign_up_activity_field_organisation_corporation": "Nom de l'entreprise",
+  "sign_up_activity_corporation_created_title": "Mon entreprise est créée",
+  "sign_up_activity_corporation_not_created_title": "Mon entreprise est en cours de création",
+  "sign_up_activity_corporation_not_created_description": "Je remplirai les infos à la création de mon entreprise",
+  "sign_up_activity_corporation_created_description": "J'ai mon n° SIRET/SIREN",
```
Comment (ghyenne) [inline packages/manager/modules/sign-up/src/form/activity/translations/Messages_fr_FR.json:15]:
```suggestion
  "sign_up_activity_corporation_created_true_title": "Mon entreprise est créée",
  "sign_up_activity_corporation_created_false_title": "Mon entreprise est en cours de création",
  "sign_up_activity_corporation_created_true_description": "Je remplirai les infos à la création de mon entreprise",
  "sign_up_activity_corporation_created_false_description": "J'ai mon n° SIRET/SIREN",
```

**Review by darsene — APPROVED** [review]

**Review by qpavy — DISMISSED** [review]

File: packages/manager/modules/sign-up/src/form/activity/controller.js [hunk 3]
```diff
@@ -47,6 +48,13 @@ export default class OvhSignUpActivityCtrl {
     }
   }
 
+  resetCorporationData() {
+    this.signUpFormCtrl.model.companyNationalIdentificationNumber = null;
+    this.signUpFormCtrl.model.organisation = null;
+    this.signUpFormCtrl.model.corporationType = null;
+    this.signUpFormCtrl.model.vat = null;
+  }
```
Comment (anooparveti) [inline packages/manager/modules/sign-up/src/form/activity/controller.js:56]: **question:** Where is this function used?

**Review by anooparveti — APPROVED** [review]

File: packages/manager/modules/sign-up/src/form/activity/controller.js (hunk 3 above)
Comment (anooparveti) [inline packages/manager/modules/sign-up/src/form/activity/controller.js:56]: **question:** Where is this function used?

**Review by anooparveti — CHANGES_REQUESTED** [review]

File: packages/manager/modules/sign-up/src/form/activity/translations/Messages_fr_FR.json [hunk 4]
```diff
@@ -9,6 +9,10 @@
   "sign_up_activity_field_vat_personalcorporation_more": "Numéro d'identification d'entreprise ({{ vatLabel }})",
   "sign_up_activity_field_organisation_association": "Nom de l'association",
   "sign_up_activity_field_organisation_corporation": "Nom de l'entreprise",
+  "sign_up_activity_corporation_created_true_title": "Mon entreprise est créée",
+  "sign_up_activity_corporation_created_false_title": "Mon entreprise est en cours de création",
+  "sign_up_activity_corporation_created_true_description": "Je remplirai les infos à la création de mon entreprise",
+  "sign_up_activity_corporation_created_false_description": "J'ai mon n° SIRET/SIREN",
```
Comment (anooparveti) [inline packages/manager/modules/sign-up/src/form/activity/translations/Messages_fr_FR.json:15]: **nitpick:** `true` and `false` in the message keys doesn't make sense to me. Better to use `created` and `not_created`

Comment (dectotam) [reply packages/manager/modules/sign-up/src/form/activity/translations/Messages_fr_FR.json:15]: This was a guillaume's suggestion so only one select-picker is created. Should I go back to the previous state (two select pickers with a ng-if and the created / not _created labels) ?

File: packages/manager/modules/sign-up/src/form/activity/controller.js (hunk 3 above)
Comment (dectotam) [reply packages/manager/modules/sign-up/src/form/activity/controller.js:56]: In the template but it was lost in the refacto... Thx, I'll add it again

File: packages/manager/modules/sign-up/src/form/activity/translations/Messages_fr_FR.json (hunk 4 above)
Comment (anooparveti) [reply packages/manager/modules/sign-up/src/form/activity/translations/Messages_fr_FR.json:15]: Yes I think it is better for better readability

**Review by darsene — APPROVED** [review]

**Review by qpavy — APPROVED** [review]

**Review by anooparveti — APPROVED** [review]

**Pull request merged**